package.json

```diff
@@ -4,14 +4,13 @@
   "description": "A package to act as a starter for typescript lambdas",
   "main": "index.js",
   "scripts": {
+    "start": "sam local start-api",
     "lint": "eslint '*/**/*.ts' --quiet --fix",
     "lint:ci": "eslint '*/**/*.ts'",
     "test": "jest --coverage",
     "test:ci": "jest --ci",
-    "build:dev": "NODE_ENV=development webpack-cli --config webpack/webpack.development.js",
-    "watch:dev": "NODE_ENV=development webpack-cli --config webpack/webpack.development.watch.js",
```

Comment:
What is the motivation for removing webpack ?

Reply:
I can't remember the specifics, but it was putting the published package in a place that the CVS pipelines didn't like. I did plan on revisiting this as the alternative is a bit ugly.